Description: fdkaac ins in Debian contrib, unfortunately. Can't have it linked to by Debian's OpenBoard.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/src/podcast/podcast.pri
+++ b/src/podcast/podcast.pri
@@ -67,7 +67,7 @@
             -lva \
             -lxcb-shm \
             -lxcb-xfixes \
-            -lxcb-render -lxcb-shape -lxcb -lX11 -lasound -lSDL -lx264 -lpthread -lvpx -lvorbisenc -lvorbis -ltheoraenc -ltheoradec -logg -lopus -lmp3lame -lfreetype -lfdk-aac -lass -llzma -lbz2 -lz -ldl -lswresample -lswscale -lavutil -lm
+            -lxcb-render -lxcb-shape -lxcb -lX11 -lasound -lSDL -lx264 -lpthread -lvpx -lvorbisenc -lvorbis -ltheoraenc -ltheoradec -logg -lopus -lmp3lame -lfreetype -lass -llzma -lbz2 -lz -ldl -lswresample -lswscale -lavutil -lm
 
     FFMPEG_VERSION = $$system(ffmpeg --version|& grep -oP "version.*?\K[0-9]\.[0-9]")
     equals(FFMPEG_VERSION, 2.8) {
